test/ctia/task/rollover_test.clj

```diff
@@ -105,3 +106,6 @@
             (is (every? nil? [ko-type-1 ko-type-2]))
             (is (every? (fn [k] (some? (:rolled_over k)))
                         [ok-type-1 ok-type-2 ok-type-3]))))))))
+
+(deftest mk-app!-test
+  (is (map? (app/service-graph (sut/mk-app!)))))
```

Comment:
There is a predicate [service-graph?](https://github.com/puppetlabs/trapperkeeper/blob/3e5e7e286287d75e7fdf7eb1dabb2fa534091329/src/puppetlabs/trapperkeeper/internal.clj#L39-L46) defined in trapperkeeper. I don't think it adds any difference to the test but it could make it a little bit more explicit, wdyt?

Also, is this test suppose to test that no exception is thrown during building an app?

Reply:
I was about to add it, but the namespace is `puppetlabs.trapperkeeper.internal`, I am not keen on using functions that are not meant for external use.
For your question, I quickly wrapped it to avoid regressions and check that we actually return a trapper keeper app with valid services.